ella_flatcomments/views.py

```diff
@@ -73,11 +73,17 @@ def post_comment(request, context, comment_id=None):
         if comment.user != request.user and not comments_settings.IS_MODERATOR_FUNC(request.user):
             return HttpResponseForbidden("You cannot edit other people's comments.")
 
+        # Don't allow users to edit a comment after the allowed edit time has expired
+        if comment.user == request.user and not comments_settings.IS_MODERATOR_FUNC(request.user) and comment.is_edit_timer_expired():
```

Comment:
this is confusing, you can just say:

    if not comments_settings.IS_MODERATOR_FUNC(request.user) and comment.is_edit_timer_expired():

also this will fail if it's not done via ajax for no reason, why? Also when it fails because it's not via ajax it reponses with 403 and "allowed time" message, that is misleading